mm2src/mm2_core/src/mm_ctx.rs

```diff
@@ -128,6 +130,9 @@ pub struct MmCtx {
     pub db_namespace: DbNamespaceId,
     /// The context belonging to the `nft` mod: `NftCtx`.
     pub nft_ctx: Mutex<Option<Arc<dyn Any + 'static + Send + Sync>>>,
+    /// asynchronous handle for rusqlite connection.
+    #[cfg(not(target_arch = "wasm32"))]
+    pub async_sqlite_connection: Constructible<Arc<AsyncMutex<AsyncConnection>>>,
```

Comment:
Maybe add a comment above these lines https://github.com/KomodoPlatform/komodo-defi-framework/blob/6cc22bca8d2f8e27ee9a7b4bc8df18847db76676/mm2src/mm2_core/src/mm_ctx.rs#L115-L118 that they are deprecated and we should use async connection for new implementations.

Reply:
Sure I will add comms. 
But could you please clarify the moment, I see that we use `shared_sqlite_conn` for `hd_wallet_storage`, should I also add `shared_async_sqlite_conn`?

Reply:
Done

Reply:
> should I also add shared_async_sqlite_conn?

Not needed for now, we will add whenever we need it.

Reply:
For the comment above `shared_sqlite_conn`, please change it to remind anyone to add `shared_async_sqlite_conn` for new implementations.

Reply:
Added doc com 
```
Deprecated, please create `shared_async_sqlite_conn` for new implementations and call db `KOMODEFI-shared.db`.
```